src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -163,10 +163,10 @@ private AdsDataProvider() {
   @DataProvider
   public static Object[][] filePages() {
     return new Object[][]{
-        {"adtest", "File:Zaznaczenie 032.png", "126608052", "wka.ent/_adtest//file",
+        {"adtest", "File:Zaznaczenie 032.png", "126608052", "wka.life/_adtest//file",
          "TOP_LEADERBOARD", "TOP_RIGHT_BOXAD", new Dimension(1292, 1000)},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "TOP_RIGHT_BOXAD" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)